js/index.js

```diff
@@ -0,0 +1,21 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const footer = document.createElement('footer');
+    document.body.appendChild(footer);
+
+    let today = new Date();
+    let thisYear = today.getFullYear();
+
+    let copyright = document.createElement('p');
+    copyright.innerHTML = `&copy; ${thisYear} Corcione Consulting`;
+    footer.appendChild(copyright);
+
+    const skills = ["JavaScript", "HTML", "CSS", "GitHub"];
+    const skillsSection = document.getElementById('skills');
+    const skillsList = skillsSection.querySelector('ul');
+
+    for (let i = 0; i < skills.length; i++) {
+        let skill = document.createElement('li');
+        skill.innerText = skills[i];
+        skillsList.appendChild(skill);
+    }
```

Comment:
What do you think about using the same concept of having some data in an array and iterate over each item (like the skills section), but instead of having items as strings you could have objects for your **experience section**, and array containing [ { title: 'Content Marketing Manager', date: '2024-Present', description: 'Develops and executes stra....' } , { experience object 2 }, { experience object 3 }, etc... ]
In this way your HTML code will be more concise and easier to read and debug, and next time you have another job experience you can just add it to the array ( **This challenge is completely optional** )